### ISSUE TITLE
Failed to search inside archive 'D:\takeout-20240823T142259Z-001.zip': Execution has failed due to an internal SevenZipSharp issue (0x80131620 / -2146232800).

### ISSUE THREAD

**Issue by ronenfe** [dnGrep/dnGrep#1235]
Trying to search in a file for a text gives this error.

**Comment by doug24** [comment]
I have no way of knowing what it might be. My guess it there is something wrong with the archive. Google says the error code is an IO exception.

Can you post the archive?

**Comment by ronenfe** [comment]
It's a 10 gb takeout from Google drive.

**Comment by doug24** [comment]
Is it one part of a multi-part archive?  It could be that the parts are not individually accessible.

I doubt there is anything I can do to help with this.

**Comment by ronenfe** [comment]
No. They make several separate parts. Anyway I extracted the file. And I will search the folders regularly.
